src/mosquito/queue.cr

```diff
@@ -94,6 +97,7 @@ module Mosquito
     end
 
     def enqueue(job_run : JobRun, at execute_time : Time) : JobRun
+      Log.trace { "Enqueuing #{job_run} at #{execute_time}" }
```

Comment:
Do you want to leave this log in here?

Reply:
I think so, yes. It's a trace, so it's not going to be seen by anyone that doesn't explicitly enable it, and if you're digging that deep you probably want to know. 

I added a bunch of trace logging in this and I hope I never have to use it  :)